src/Security/Security.Autorest/examples/Get-AzSecurityApiCollection.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
+```powershell
```

Comment:
Please provide the example for the cmdlet. The same below

Reply:
Thank you for the review. I added examples for all cmdlets.